src/pages/dashboard/DashboardPage.tsx

```diff
@@ -28,6 +31,10 @@ const DashboardPage = () => {
   const [endDate, setEndDate] = useDate(now.clone().subtract(1, 'day'))
   const [groupByHour, setGroupByHour] = React.useState<boolean>(false)
 
+  const { search, setSearch } = useContext(SearchContext)
+  const { operatorId, lineNumber, timestamp, routes, routeKey } = search
```

Comment:
```suggestion
  const [operatorId, setOperatorId] = useState('')
```
will be easier.
we don't need the entire search context here

Reply:
Done